package.json

```diff
@@ -16,7 +16,7 @@
     "jquery-ujs": "^1.2.2",
     "moment": "^2.29.4",
     "stimulus": "^3.0.0",
-    "sweetalert2": "^11.7.0",
+    "sweetalert2": "^11.4.20",
```

Comment:
why are we downgrading this?

Reply:
Not sure where that came from. Fixed now.